game/graphics/opengl_renderer/OpenGLRenderer.cpp

```diff
@@ -838,35 +852,26 @@ void OpenGLRenderer::setup_frame(const RenderOptions& settings) {
     m_fbo_state.resources.render_buffer.clear();
     m_fbo_state.resources.resolve_buffer.clear();
 
-    // first, see if we can just render straight to the display framebuffer.
-    // note: we always force a separate fbo on a screenshot so that it won't capture overlays.
-    //       as an added bonus it also doesn't break the sprite distort buffer...
-    if (!settings.save_screenshot &&
-        window_fb.matches(settings.game_res_w, settings.game_res_h, settings.msaa_samples)) {
-      // it matches - no need for extra framebuffers.
-      lg::info("FBO Setup: rendering directly to window framebuffer");
-      m_fbo_state.render_fbo = &m_fbo_state.resources.window;
-    } else {
-      lg::info("FBO Setup: window didn't match: {} {}", window_fb.width, window_fb.height);
+    // NOTE: we will ALWAYS render the game to a separate framebuffer instead of directly to the
+    // window framebuffer.
```

Comment:
thanks, I've wanted to do this for a long time. I can simplify some of the sprite-glow stuff after this.